app/src/main/java/com/rob729/quiethours/Fragments/NewProfileFragment.kt

```diff
@@ -161,15 +158,15 @@ class NewProfileFragment : Fragment() {
                 var i = 0
                 while (i < 7) {
                     if (days[i]) {
-                        sAlarm(i + 1, profile)
+                        Utils.sAlarm(requireContext(), i + 1, profile, false)
```

Comment:
Rename the sAlarm and eAlarm to setStartAlarm and setEndAlarm for better understanding,

Reply:
@rob729 Done